src/tabs/map.rs

```diff
@@ -57,6 +57,156 @@ impl Tab {
             force_close: false,
         })
     }
+
+    fn recompute_autotile(&self, map: &rpg::Map, position: (usize, usize, usize)) -> i16 {
```

Comment:
Woah, how did you figure this out?

Reply:
I looked at how RPG Maker XP behaves when you place autotiles and used how the autotile variants are ordered numerically to deduce this algorithm